tests/snmpsim/siteboss571_siteboss571.snmprec

```diff
@@ -1,6 +1,6 @@
-1.3.6.1.2.1.1.1.0|4|\"SiteBoss 571 2.11.380 STD
```

Comment:
Hi @jediblair 
Could you please create a new snmprec file instead of changing the existing ?